flowmachine/flowmachine/features/subscriber/subscriber_tacs.py

```diff
@@ -380,9 +452,10 @@ class SubscriberPhoneType(SubscriberFeature):
     Examples
     -------------
 
-    >>> subscriber_smart = SubscriberPhoneType('2016-01-01', '2016-01-07')
+    >>> subscriber_smart = SubscriberHandsetCharacteristic('2016-01-01', '2016-01-07',
+        'hnd_type')
     >>> subscriber_smart.get_dataframe()
-             subscriber handset_type
+             subscriber     hnd_type
```

Comment:
This should use `value` as the generic column name please. (See https://github.com/Flowminder/FlowKit/issues/421)

Reply:
Ah, I just saw that this is already implemented below so this was just a leftover in the docstring. 👍